models/articles-model.js

```diff
@@ -6,6 +6,21 @@ exports.fetchArticleById = async (article_id) => {
     if (body.rows.length === 0) {
         return Promise.reject({ status: 404, message: "article id not found"})
     } else {
-        return body.rows
-    }        
-}
\ No newline at end of file
+        return body.rows[0]  
+}}
+
+exports.fetchArticles = async () => {
+    const query = `
+    SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count
+    FROM articles
```

Comment:
If you wanted to look into getting your comments back as a number you could look at the SQL CAST Function.

https://www.w3schools.com/sql/func_sqlserver_cast.asp